script/Ciel/Day 12/12_01_CIEL12_3_2B.txt

```diff
@@ -1,72 +1,82 @@
 -- もう会えない訳じゃないんだ。今は無理をしない方がいい。
-It's not like I'll never be able to see her again. I shouldn't be pushing myself right now.
+It's not like I'll never get another chance to see those two again.
+ I should just take things easy for now.
 }
 [sha:ed1171ae8065cd5d4e92a6ad2fe5d6bde4d1afe9]{
 -- Page 194, Offset 31401.
 -- 「……そう、だね……今日は、もう」
-"...You're, right... I think today's..."
+"Yeah... you're right.
+ I'll call it a day."
 }
 [sha:4cbcd6de1d307ae8f68fc5b9b380c116c0cba258]{
 -- Page 194, Offset 31402.
 -- 「よかった、お休みいただけるんですね。
-"Thank goodness, you're going to rest now, aren't you?
+"Thank goodness. You'll go rest now, won't you?
 }
 [sha:96a78701e8898ff00bbeda30a76c2b1d5745a199]{
 -- Page 194, Offset 31403.
 -- ……志貴さん、おひとりで歩けますか？」
-...Shiki-san, can you walk on your own?"
+Shiki-san, can you walk on your own?"
 }
 [sha:0b5de615cbe1254e3e8a472169486efcf9fa4014]{
 -- Page 194, Offset 31404.
 -- 「っ……いや、ごめん」
-"...Sorry, I can't."
+// っ silent
+"I can't, sorry..."
 }
 [sha:50f3bb100179d0b982d7b916a4773ff1ae43d58c]{
 -- Page 194, Offset 31405.
 -- どうにも、立っている事さえ無理そうだ。
-I don't think I can even stand.
+In fact, I can barely even stand.
 }
 [sha:6c1b2769ca7bb90828445c055908abf0593634aa]{
 -- Page 195, Offset 31406.
 -- 「あ、ちょうど良かった翡翠ちゃん！　志貴さんをお部屋までお連れして！」
-"Ah, you're right on time, Hisui-chan! Go help Shiki-san up to his room!"
+"Ah, perfect timing, Hisui-chan!
+ Go help Shiki-san back to his room!"
 }
 [sha:63ba435df1e512fafc33b154ea530d1fb5250fdd]{
 -- Page 195, Offset 31407.
 -- 「ぁ―――姉さん、ですが、わたしは―――」
-"Ah―――Nee-san, but I―――"
+"Ah... But, Nee-san, I―――"
 }
 [sha:33fb4959408f1af493dbc3f640b4d01eec85e459]{
 -- Page 196, Offset 31408.
 -- 「翡翠ちゃん！　志貴さんは病人なんですよ、困っている人を見捨てるのは貴女の性格じゃないでしょう？
-"Hisui-chan! Shiki-san is hurt right now. It isn't like you to abandon a person in need, is it?
+"Hisui-chan! Shiki-san is hurt right now.
+ You aren't the sort of person to abandon someone in need, are you?
 }
 [sha:e8454132dee07c0106ffe02b81b28fa50e317d9c]{
 -- Page 196, Offset 31409.
 -- ええ、今日ぐらいはわたしが許します。ちゃんと志貴さんを看病するべきです！」
+// Don't love this rendering of ええ
 Right, I'll permit it just for today. You should be taking proper care of Shiki-san!"
 }
 [sha:e60938d837a63dcf9a51fa255cd8b731935c1019]{
 -- Page 197, Offset 31410.
 -- 「……はい。ごめんなさい姉さん、わがままを言って」
-"...Of course. I'm sorry, Nee-san. That was selfish of me."
+"Of course... I'm sorry, Nee-san. That was selfish of me."
 }
 [sha:1bc4df92019ec7d5782786156a63dfc700984aed]{
 -- Page 197, Offset 31411.
 -- 「わかってくれればいいんです。それでお薬は見つかった？」
-"It's okay as long as you understand. So did you find the medicine?"
+"It's okay as long as you understand. So, did you find the medicine?"
 }
 [sha:6c5b00b314ea0046824ad7ef4e6d1337228a3e05]{
 -- Page 198, Offset 31412.
 -- 「……その、姉さんの部屋の薬箱は多すぎて。解熱剤はこれ、でしょうか？」
-"...Well, there were an awful lot of medicine boxes in your room. Is this the antipyretic you asked for?"
+"Well... there were an awful lot of medicine boxes in your room.
+ Is this the antipyretic you asked for?"
```

Comment:
"an awful lot of" somehow doesn't seem very Hisui. 
Should probably lead in the second sentence to match the above one a bit better. I think the "but" does a lot of work, but it could be unnecessary.
Oh, and 'antipyretic' feels a tinge too sciency, but that kind of seems like the point? Kinda part of why she's unsure.
>Well... the medicine boxes in your room were far too numerous, but...
>Could this be the 'antipyretic' you asked for?

Reply:
Yeah the problem with antipyretic is that people colloquially just refer to it as a brand name (paracetamol, tylenol) or as an NSAID, none of which are really better. Maybe just 'medication'?